src/rtl/xdprtl.c

```diff
@@ -5,6 +5,10 @@
 
 #include "precomp.h"
 
+EX_RUNDOWN_REF XdpRtlRundown = {
+    .Count = EX_RUNDOWN_ACTIVE
```

Comment:
`ExReInitializeRundownProtection` below sets this when `XdpRtlStart` is called, right? Why not initialize this to zero instead, which would more closely match what happens after you've been stopped (but not unloaded) and restarted once.

Reply:
This is the right way - (re)initializing a rundown clears the flag and allows new references. We don't want to allow new references until the library is initialized or after it is stopped.

Reply:
RUNDOWN_ACTIVE means new rundown references are disabled, which might be the cause of confusion here.

Reply:
Ah, you're right. I just checked our MsQuic code and it does the same thing (inside the abstraction layer). Another comment might be nice to say "ACTIVE means disabled" because that is confusing from a code inspection without more context.

Reply:
IMO familiarity with Windows APIs is table stakes for this project - it's not our job to document the APIs throughout our codebase.